lib/jxl/splines.cc

```diff
@@ -634,6 +645,19 @@ Status Splines::InitializeDrawCache(const size_t image_xsize,
       return JXL_FAILURE("Too many pixels covered with splines");
     }
   }
+  if (total_estimated_area_reached >
+      std::min((image_xsize * image_ysize + (uint64_t(1) << 18)),
+               (uint64_t(1) << 22))) {
+    JXL_WARNING(
```

Comment:
TODO about making this a hard failure for L5 bitstreams?

Reply:
done